documentation/domains/Cluster.json

```diff
@@ -11,7 +11,7 @@
           "$ref": "#/definitions/ServerService"
         },
         "serverPod": {
-          "description": "Customization affecting the generation of Pods for WebLogic Server instances.",
+          "description": "Customization affecting the generation of Pods for WebLogic Server instances or Introspector Job Pod.",
```

Comment:
Cluster level serverPod settings are not expected to have any effect on the introspector pod.

Reply:
There's a general issue with Cluster.json where all domain level settings are included in Cluster.json. This needs to be investigated separately.

Reply:
Fixed in [a6b2383](https://github.com/oracle/weblogic-kubernetes-operator/pull/3914/commits/a6b23837e674e333f027e573edba8c9dedd57ca5).